app/build.gradle

```diff
@@ -86,21 +86,22 @@ android {
 ext {
     supportLibVersion = '28.0.0'
```

Comment:
Since there's no need for this now, could you please remove?

Reply:
Done.